cache_invalidator/Makefile

```diff
@@ -0,0 +1,9 @@
+CC=gcc
```

Comment:
This should probably be `CC ?= gcc` if you want to support other compilers. If you don't want to support other compilers, then you may as well just hard-code it instead of using `${CC}`